### ISSUE TITLE
Why the new Google analytics code was reverted?

### ISSUE THREAD

**Issue by secumod** [pages-themes/cayman#48]
The is no description in d2f951c as to why the `gtag` Google analytics code was reverted. Could you, please, clarify?

**Comment by benbalter** [comment]
@sjparkinson Looks like this was part of https://github.com/pages-themes/cayman/pull/43. Can you explain your reasoning?

**Comment by sjparkinson** [comment]
https://github.com/pages-themes/cayman/pull/44

**Comment by sjparkinson** [comment]
It was separated out into a new PR.

**Comment by sjparkinson** [comment]
I think we've hit a bad merge by doing #43 after #44. Reverting the revert should correct it.

I should have cleaned up #43 better after the suggestion to split the work, my mistake.

**Comment by benbalter** [comment]
@secumod if you'd like to submit a pull request to add it back, I'd be glad to merge it.
